website/docs/kubernetes/viewing-and-selecting-current-kubernete-context-in-the-status-bar.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+title: Selecting a context in the status bar
+description: Viewing and selecting the current Kubernetes context in the status bar
+keywords: [podman desktop, podman, containers, pods, migrating, kubernetes]
+tags: [migrating-to-kubernetes]
+---
+
+# Viewing and selecting the current Kubernetes context
+
+With Podman Desktop, you can view and select your current Kubernetes context in the status bar.
```

Comment:
kubernetes context = what is defined in $HOME/.kube/config or a custom location in preferences of Podman Desktop